src/parser/jobs/blu/modules/GeneralCDDowntime.ts

```diff
@@ -1,9 +1,27 @@
+import {Events} from 'event'
+import {dependency} from 'parser/core/Injectable'
+import {Actors} from 'parser/core/modules/Actors'
 import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
 const DEFAULT_ALLOWED_DOWNTIME = 1000
+const MAX_WINGED_STACKS_BEFORE_CD = 3
 export class GeneralCDDowntime extends CooldownDowntime {
 	override defaultAllowedAverageDowntime = DEFAULT_ALLOWED_DOWNTIME
 
+	@dependency private actors!: Actors
+
+	override countUsage(e: Events['action']): boolean {
+		const actionId = e.action
+		if (actionId !== this.data.actions.WINGED_REPROBATION.id) {
+			return true
+		}
+		const wingedReprobationStacks = this.actors.current.getStatusData(this.data.statuses.WINGED_REPROBATION.id) ?? 0
```

Comment:
Do we need this countUsage override if we're not tracking Winged Reprobation? Alternatively, should we be tracking it? If the latter, that's definitely going to need patch level support (and can be a later PR once we get someone to merge the one adding the patch information...)

Reply:
It should be tracked, but Winged is also optional for some new types of openers so I need to do more work before we start telling people about missing uses.

Reply:
Fair enough. Since we're planning to track it eventually, I'm fine with leaving the code in.

Reply:
Rebased the branch & included a changelog entry!